src/entropy.jl

```diff
@@ -57,19 +56,20 @@ abstract type EntropyEstimator <: AbstractEntropy end
 ###########################################################################################
 # Notice that StatsBase.jl exports `entropy` and Wavelets.jl exports `Entropy`.
 """
-    entropy([e::Entropy,] probs::Probabilities) → h::Real ∈ [0, ∞)
-    entropy([e::Entropy,] est::ProbabilitiesEstimator, x) → h::Real ∈ [0, ∞)
-    entropy([e::Entropy,] est::EntropyEstimator, x) → h::Real ∈ [0, ∞)
+    entropy([e::Entropy,] probs::Probabilities)
+    entropy([e::Entropy,] est::ProbabilitiesEstimator, x)
+    entropy([e::Entropy,] est::EntropyEstimator, x)
 
-Compute `h`, a (generalized) [`Entropy`](@ref) of type `e`, in one of three ways:
+Compute `h`, with `h::Real ∈ [0, ∞)`, which is
```

Comment:
Ah, I just noticed. This actually not true for `EntropyEstimator`s, because they estimate differential (continuous) entropy, i.e. an integral. The differential entropy can actually be negative (trivial example: entropy of a uniform distribution from [0, 1/2]). 

For discrete estimators, the entropy is always positive, though. We should distinguish between these cases in the docs.